api/v1beta1/database_webhook.go

```diff
@@ -54,15 +58,40 @@ var _ webhook.Validator = &Database{}
 func (r *Database) ValidateCreate() error {
 	databaselog.Info("validate create", "name", r.Name)
 
-	// TODO(user): fill in your validation logic upon object creation.
+	err := validateSecretTemplates(r.Spec.SecretsTemplates); if err != nil {
+		return err
+	}
+	
 	return nil
 }
 
 // ValidateUpdate implements webhook.Validator so a webhook will be registered for the type
 func (r *Database) ValidateUpdate(old runtime.Object) error {
 	databaselog.Info("validate update", "name", r.Name)
 
-	// TODO(user): fill in your validation logic upon object update.
+	err := validateSecretTemplates(r.Spec.SecretsTemplates); if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func validateSecretTemplates(templates map[string]string) error {
```

Comment:
would like to have a test for this function 🙃

Reply:
Test is added, and I've also updated validation rules, so now I'm not removing dots